.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: required
+
+services:
+  - docker
+
+language: node_js
+node_js:
+  - "8"
+
+env:
+  - TRUFFLE_DEVELOP_HOST=truffle TRUFFLE_DEVELOP_PORT=9545'
```

Comment:
The last single quote is not needed, or?

Reply:
I checked that it works even with that https://travis-ci.org/cryptomental/MarketProtocol/jobs/313547119